src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ViewDataDictionary.cs

```diff
@@ -60,7 +67,7 @@ public ViewDataDictionary(ViewDataDictionary source)
         /// <remarks>
         /// For use when the new instance's declared <see cref="Model"/> <see cref="Type"/> is unknown but its
         /// <see cref="Model"/> is known. In this case, <see cref="object"/> is the best possible guess about the
-        /// declared type when <paramref name="model"/> is <c>null</c>.
+        /// declared type.
         /// </remarks>
         public ViewDataDictionary(ViewDataDictionary source, object model)
```

Comment:
@Eilon this constructor is almost identical to calling `new ViewDataDictionary<object>(source, model)` but it's less explicit. The only product code using it is within `@Html.PartialAsync()`. Any objection to removing it?